pyproject.toml

```diff
@@ -5,31 +5,32 @@ description = ""
 authors = [
     "UnitaryFund",
     "Daniel Strano <dan@unitary.fund>",
-    "Vincent Russo <vincent@unitary.fund>"
+    "Vincent Russo <vincent@unitary.fund>",
+    "Alessandro Cosentino <alessandro@unitary.fund>"
```

Comment:
Sorry, I was clearly trying to take all the glory for this. Good call!

Reply:
technically I was already in the «UnitaryFund» author :)